flang/lib/Optimizer/Dialect/Support/FIRContext.cpp

```diff
@@ -88,6 +88,36 @@ void fir::setTuneCPU(mlir::ModuleOp mod, llvm::StringRef cpu) {
   mod->setAttr(tuneCpuName, mlir::StringAttr::get(ctx, cpu));
 }
 
+static constexpr const char *ignoreDenormalModeName =
+    "fir.ignore.denormal.mode";
```

Comment:
Nit: Generally, dots are used to separate "scopes" in some hierarchy. I'd suggest following the `<dialect> <dot> <option name>` convention here.
```suggestion
    "fir.atomic_ignore_denormal_mode";
```